h2o-bindings/bin/gen_R.py

```diff
@@ -284,6 +284,8 @@ def get_help(param, indent=0):
     phelp = param.get('help')
     if not phelp:
         return
+    # fix braces so that CRAN doesn't complain
+    phelp = phelp.replace("{", "\{").replace("}", "\}")
     if ptype == 'boolean':
```

Comment:
This should already be in both master and rel-3.44.0. https://github.com/h2oai/h2o-3/pull/16010/files#diff-231f2d963f774362ad0fb01f7825ddc8cf1f32d59ce0c5dd053abea9402b055fR287-R288

I'm confused why I see the change here. Current master already has that (https://github.com/h2oai/h2o-3/blob/master/h2o-bindings/bin/gen_R.py#L287-L288),